cishouseholds/merge.py

```diff
@@ -5,6 +5,23 @@
 from pyspark.sql import functions as F
 from pyspark.sql.window import Window
 
+from cishouseholds.compare import prepare_for_union
+
+
+def merge_survey_tables(tables: List[Union[str, DataFrame]]):
```

Comment:
A user could pass more or less than 3 DFs here, so would be safer to have three explicit arguments here